src/Tizen.TV.UIControls.Forms/Renderer/MediaSourceHandler.cs

```diff
@@ -19,41 +19,77 @@
 using TFileMediaSource = Tizen.TV.UIControls.Forms.FileMediaSource;
+using System.Collections.Generic;
 
+[assembly: ExportHandler(typeof(DRMMediaSource), typeof(DRMMediaSourceHandler))]
 [assembly: ExportHandler(typeof(TUriMediaSource), typeof(UriMediaSourceHandler))]
 [assembly: ExportHandler(typeof(TFileMediaSource), typeof(FileMediaSourceHandler))]
 namespace Tizen.TV.UIControls.Forms.Renderer
 {
 
     public interface IMediaSourceHandler : IRegisterable
     {
-        Task<bool> SetSource(Player player, MediaSource imageSource);
+        Task<bool> SetSource(MediaPlayerImpl player, MediaSource imageSource);
     }
 
     public sealed class UriMediaSourceHandler : IMediaSourceHandler
     {
-        public Task<bool> SetSource(Player player, MediaSource source)
+        public Task<bool> SetSource(MediaPlayerImpl player, MediaSource source)
         {
             if (source is UriMediaSource uriSource)
             {
                 Log.Info(UIControls.Tag, $"Set UriMediaSource");
                 var uri = uriSource.Uri;
-                player.SetSource(new MediaUriSource(uri.IsFile ? uri.LocalPath : uri.AbsoluteUri));
+                player.NativePlayer.SetSource(new MediaUriSource(uri.IsFile ? uri.LocalPath : uri.AbsoluteUri));
             }
             return Task.FromResult<bool>(true);
         }
     }
 
     public sealed class FileMediaSourceHandler : IMediaSourceHandler
     {
-        public Task<bool> SetSource(Player player, MediaSource source)
+        public Task<bool> SetSource(MediaPlayerImpl player, MediaSource source)
         {
             if (source is FileMediaSource fileSource)
             {
                 Log.Info(UIControls.Tag, $"Set FileMediaSource");
-                player.SetSource(new MediaUriSource(ResourcePath.GetPath(fileSource.File)));
+                player.NativePlayer.SetSource(new MediaUriSource(ResourcePath.GetPath(fileSource.File)));
+            }
+            return Task.FromResult<bool>(true);
+        }
+    }
+    public sealed class DRMMediaSourceHandler : IMediaSourceHandler
+    {
+        public Task<bool> SetSource(MediaPlayerImpl player, TV.UIControls.Forms.MediaSource source)
+        {
+            if (source is DRMMediaSource uriSource)
+            {
+                var platformDrmMgr = player.DRMManager;
+                if (player.IsDRMOpened == true)
+                {
+                    player.IsDRMOpened = false;
+                    platformDrmMgr.Close();
+                }
+                var appId = Tizen.Applications.Application.Current.ApplicationInfo.ApplicationId;
+                platformDrmMgr.Init(appId);
+                platformDrmMgr.AddProperty("LicenseServer", uriSource.LicenseUrl);
+                foreach (KeyValuePair<string, ExtradataValue> pair in uriSource.Extradata)
+                {
+                    platformDrmMgr.RemoveProperty(pair.Key);
+                    platformDrmMgr.AddProperty(pair.Key, pair.Value.Value);
+                }
```

Comment:
If platformDrmManager does not support overwrite about duplicated key, 
You also remove `LicenseServer` propperty before add `LicenseServer` property